main.py

```diff
@@ -2,6 +2,54 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+
+    genres = ["Western", "Action", "Dramma"]
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+
+    for first, last in actors:
+        Actor.objects.create(
+            first_name=first,
+            last_name=last
+        )
+
+    Genre.objects.filter(
+        name="Dramma"
+    ).update(name="Drama")
+
+    Actor.objects.filter(
+        last_name="Klooney"
+    ).update(last_name="Clooney")
```

Comment:
Asking to change the surname of a particular actor, not all actors with that surname